marathon_lb.py

```diff
@@ -1208,6 +1208,8 @@ def compareMapFile(map_file, map_string):
 
 
 def get_health_check(app, portIndex):
+    if 'healthChecks' not in app:
```

Comment:
Because healthChecks may not be there which was causing a KeyError when this function was called without ensuring that 'healthChecks' was present.